recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "docformatter" %}
-{% set version = "1.4" %}
+{% set version = "1.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 064e6d81f04ac96bc0d176cbaae953a0332482b22d3ad70d47c8a7f2732eef6f
+  sha256: 9dc71659d3b853c3018cd7b2ec34d5d054370128e12b79ee655498cb339cc711
 
 build:
```

Comment:
Build is failing with `ModuleNotFoundError: No module named 'poetry'` at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=554671&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=206 because `docformatter` not uses poetry for build (see https://github.com/PyCQA/docformatter/blob/1def3c413021cdc0d355eb988444fbc847181173/pyproject.toml#L87). Could you try adding `poetry` to the host requirements at L22, specifically a line like:

```
    - poetry-core >=1.0.0
```

Reply:
thanks for point this out, and sorry for the late reply. If you want, I can add you as maintainer if you are interested as an active user of this package.

Reply:
I'm not actually a very active docformatter user, but happy to be added as a recipe maintainer to help out. Will open a separate issue to add myself in 😊